src/lib/ConnectWallet.svelte

```diff
@@ -22,7 +22,7 @@
     <Card>
         <GradientTitle marginTop="1em" marginBottom="0.5em">Nahmii Bridge</GradientTitle>
         <p>
-            Welcome to the entrance of the most powerful Ethereum scaling solution, paving the way for unprecedented Blockchain based user experiences. Use its power at your own risk. 
+            Welcome to the entrance of the most powerful Ethereum scaling solution, paving the way for unprecedented Blockchain based user experiences. Wield its power with responsibility. 
```

Comment:
I suggest the latter sentence about responsibility be replaced by "Cautious exercise of power is advised."